### PR TITLE
New Rakefile

### PR DESCRIPTION
**Pull request by theborch**
Fixes some issues with rubocop and cleans up the integration tasks.

### CODE REVIEW

**Comment by jujugrrr** [comment]
@theborch have we done this on all the Stacks? We need to be sure Stack stay aligned, they should all have the same file

File: Rakefile [hunk 1]
```diff
@@ -1,64 +1,57 @@
...
-begin
-  require 'kitchen/rake_tasks'
-  Kitchen::RakeTasks.new
-  rescue LoadError
-    puts '>>>>> Kitchen gem not loaded, omitting tasks' unless ENV['CI']
+  desc 'Run Test Kitchen with cloud plugins'
+  task :cloud do
+    if ENV['CI'] == 'true'
+      Kitchen.logger = Kitchen.default_file_logger
+      @loader = Kitchen::Loader::YAML.new(local_config: '/var/lib/jenkins/.kitchen/config.yml')
```
Comment (jujugrrr) [inline Rakefile]: Why do we have hard coded path to Jenkins here?

**Comment by jujugrrr** [comment]
We need more context on PR, especially when it's refactoring. I don't understand why it's better (I don't say it's not).

**Comment by martinb3** [comment]
@jujugrrr It unbreaks test-kitchen from the recent test-kitchen release that broke all our builds. @theborch was going to do it on every stack.

**Pull request merged**